.gitignore

```diff
@@ -87,3 +87,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# Jetbrains project files
+.idea/
```

Comment:
Why? I don't think so. Better then accidentally adding IDE's files all over the git project.